kubernetes-model-generator/kubernetes-model-common/pom.xml

```diff
@@ -32,6 +32,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
+      <version>${jackson.databind.version}</version>
```

Comment:
We seem to be using Jackson Bom dependency here:

https://github.com/fabric8io/kubernetes-client/blob/af21ba46b7f4f79d6396f9d9a30116b41270877d/pom.xml#L692-L696

Shall we update `jackson.version` property to 2.13.2.2 instead of adding another property specifically for jackson-databind?

Reply:
There's no 2.13.2.2 version for the bom. The version is [`2.13.2.20220328`](https://search.maven.org/artifact/com.fasterxml.jackson/jackson-bom).

However, this version is not aligned with the rest of the artifacts, so the karaf bundle will fail.

I'll add this PR to the 6.0 milestone and depending on the status of Jackson before the release we can proceed with these changes or evaluate other options.

Reply:
@manusa indeed there's no version 2.13.2.2 in the BOM, and probably you won't find it. Jackson Databind and the BOM have different versions due to the micro release to fix the CVE.

Reply:
There's microrelease for the BOM (the one I linked).
However, there's not an aligned version for the BOM's declared dependencies (e.g. core points to 2.13.2 and databind to 2.13.2.2), which is cumbersome for the OSGI bundle. That's why I propose to delay this just until we're about to release and see if we can just point to an updated 2.13.x bom release that references both the fix and a constant Jackson dependency for each module.

Reply:
@manusa no worries, whatever works best for the project, that's fine from me. I just wanted to bring that to your attention, so your team can decide when to upgrade.